cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -74,6 +71,8 @@ var (
 	metricsAddress = flag.String("metrics-address", "", "The TCP network address where the prometheus metrics endpoint will listen (example: `:8080`). The default is empty string, which means metrics endpoint is disabled.")
 	metricsPath    = flag.String("metrics-path", "/metrics", "The HTTP path where prometheus metrics will be exposed. Default is `/metrics`.")
 
+	defaultFSType = flag.String("default-fstype", "ext4", "Specify the filesystem type of the volume. If not specified,  external-provisioner will set default as `ext4`.")
```

Comment:
Can we make this opt-in so that plugins don't need a "none" value?

Reply:
Can you add a comment here about "none"?  Otherwise driver developer won't know that they can set "none" here unless if they read the code.

Reply:
@msau42 do you mean we should allow plugin to leave this as "" instead of setting to "none" to indicate there is no default fstype?

Reply:
Correct